botocore/utils.py

```diff
@@ -48,6 +48,20 @@
 RETRYABLE_HTTP_ERRORS = (requests.Timeout, requests.ConnectionError)
 S3_ACCELERATE_WHITELIST = ['dualstack']
 
+CONTROLS = {
+    '\n': '\\n',
+    '\r': '\\r',
+    '\t': '\\t',
```

Comment:
Are these the only control characters that will get borked by sphinx? What if the string happens to contain `\a`? Or something that looks like bytes like `\xe2`?